caddyconfig/caddyfile/dispenser.go

```diff
@@ -261,6 +312,18 @@ func (d *Dispenser) RemainingArgs() []string {
 	return args
 }
 
+// RawRemainingArgs loads any more arguments (tokens on the same line,
+// retaining quotes) into a slice and returns them. Open curly brace
+// tokens also indicate the end of arguments, and the curly brace is
+// not included in the return value nor is it loaded.
+func (d *Dispenser) RawRemainingArgs() []string {
```

Comment:
Probably a dumb nit, but what about `RemainingRawArgs()`? That keeps the two related functions together in the docs... or maybe `RemainingArgsRaw()` is more pedantically accurate.

Reply:
Yeah we can do Raw suffix, that works for me